notebooks/03-query/14-sync-async-queries.ipynb

```diff
@@ -600,6 +600,49 @@
     "The result from the query covers the full timeline of data available, the ingested data available in Deep Storage plus the latest data being streamed into the table continuously."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "21a32041-e9cb-4b14-a149-b63ccd4c0386",
+   "metadata": {},
+   "source": [
+    "# Retrieving results page by page \n",
+    "The API allows the user to retrieve results a page at a time.\n",
+    "The size of each page can be controlled with the query context parameter `rowsPerPage`.\n",
+    "The `druidapi` package now incorporates the rowsPerPage parameter into the Druid API call.  \n",
```

Comment:
Suggest rephrase as

"Use the `rowsPerPage` query context parameter to control the size of the results page.

In the cell below, the page size is set by providing a parameter in the call to the Python API."

Reply:
done